wks-controller.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         operator: Exists
       containers:
       - name: controller
-        image: quay.io/wksctl/controller:latest
+        image: docker.io/weaveworks/wksctl-controller:v0.8.2-beta.7
```

Comment:
maybe use the value from config.yaml

Reply:
Yes... that is what it does... :-) See line 137 of setup.sh:
`set_wksctl_version ${config_images_wksctl}`

The `parse_yaml()` function, reads the whole config.yaml and generates a variable for each value including **_config_images_wksctl_** using the location in the yaml hierarchy to create the variable name. (e.g. "config_" + key1 + key2 + key3...) and then assigns the value.